### PR TITLE
feat: migrate to Ionic Docs theme

### PR DESCRIPTION
**Pull request by eric-horodyski**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,18 +15,20 @@
   },
   "dependencies": {
     "@code-hike/mdx": "~0.9.0",
-    "@docusaurus/core": "3.0.1",
-    "@docusaurus/preset-classic": "3.0.1",
+    "@docusaurus/core": "~3.1.0",
+    "@ionic-docs/preset-classic": "file:./ionic-docs-preset-classic-1.1.0.tgz",
```
Comment (dtarnawsky) [inline package.json:19]: This seems a bit odd, shouldn't this be reliant on a released shared package rather than a local copy?

**Review by dtarnawsky — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (eric-horodyski) [reply package.json:19]: Jared wanted to hold off on releasing until he tests in different environments, but didn't want to hold us up so he gave me a local copy.

Comment (eric-horodyski) [reply package.json]: Jared cut a build 🎉

**Review by dtarnawsky — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -15,18 +15,20 @@
...
+    "prism-react-renderer": "^2.3.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
   },
   "devDependencies": {
-    "@docusaurus/module-type-aliases": "~3.0.1"
+    "@ionic/prettier-config": "^4.0.0",
+    "prettier": "^3.1.1"
   },
+  "prettier": "@ionic/prettier-config",
```
Comment (kensodemann) [inline package.json:31]: I am very much in favor of this addition.

**Review by kensodemann — APPROVED** [review]
LGTM!
Thank-you for putting this together.

**Comment by kensodemann** [comment]
While this all LGTM, we _do_ seem to have an issue with the deploy that needs to be handled first:

```
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /home/runner/work/tutorials/tutorials/ionic-docs-theme-classic-1.1.0.tgz
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/home/runner/work/tutorials/tutorials/ionic-docs-theme-classic-1.1.0.tgz'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

**Pull request merged**